app/javascript/components/sidebar/tabs/component.js

```diff
@@ -22,18 +25,20 @@ class TabsComponent extends React.Component {
         <ul>
           {
             items.map(item => (
-              <li key={item.value}>
-                <button
-                  className={
-                    classnames({
-                      tab: true,
-                      selected: selected === item.value
-                    })
-                  }
-                  onClick={() => this.props.onSelect(item.value)}
-                >
-                  {item.label}
-                </button>
+              <li className="tab" key={item.value}>
+                <div className={classnames('tab-item', { selected: selected === item.value })}>
+                  <Icons
```

Comment:
These `Icons` component MUST be on the root of your app.  Put it somewhere you only render it once, right now it's inside a loop!!!!